### ISSUE TITLE
Flow

### ISSUE THREAD

**Issue by javier-tarazaga** [KunalKapadia/express-mongoose-es6-rest-api#395]
Hi guys, 

Thanks for the project! Really helpful to get started. Quick one though, have you consider adding "Flow" for static type checking? 

Thanks!

**Comment by osahner** [comment]
I would rather prefer Typescript.

**Comment by okonon** [comment]
Yeah typescript would be nice. @osahner does it mean all other third party libs will need type definitions?

**Comment by osahner** [comment]
No, not at all @okonon. If the lib has some sort of es5 or es6 export you are set (like lodash etc.). Or you can import anything from the lib with `import * as express from 'express';`. Last resort would be  to declare the types `declare var $:any;`.

**Comment by okonon** [comment]
Makes sense thanks

**Comment by ngmikeng** [comment]
Hi guys,
I was also working on a Typescript version and based on this great project for purpose learning Typescript, please take a look and give me some feedback when you have chance.
https://github.com/ngmikeng/mn-express-typescript-rest-starter

**Comment by osahner** [comment]
I really like what you are doing @ngmikeng !

**Comment by ngmikeng** [comment]
Thanks a lot @osahner ! :smile:
